notebooks/Causal Model Selection with the RScorer.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
I found couples of issues in the notebook, it's hard to point to the right place here, I will list one by one and point to the cell.

- cell 5: please comment out all the other exp_te functions and only leave one.
- cell 15: last four lines should be under the for loop, effect label should change to something like 'effect estimate'
- cell 17: effect label should change to something like 'effect estimate'
- cell 27-29: cell 27-28 returns nan, cell 29 last model should be best.effect(), and the xticks is also wrong

Reply:
Done